cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-connect.php

```diff
@@ -282,7 +282,12 @@ public function setup() {
 			if ( empty( $stats ) ) {
 				// Get users plan.
 				$stats = $this->plugin->components['connect']->api->usage();
-				if ( ! is_wp_error( $stats ) && ! empty( $stats['media_limits'] ) ) {
+
+				if ( is_wp_error( $stats ) ) {
+					return;
+				}
+
+				if ( ! empty( $stats['media_limits'] ) ) {
```

Comment:
I think it might be a better idea to set some defaults before returning if there's a `WP_Error` since there are 5 uses of its keys around the plugin, which might cause `undefined index` errors.

- See `class-media.php` at line 1079-1080
- See `class-push-sync.php` at line 340-341
- See `class-upload-sync.php` at line 120

What do you think @kienstra ?

Reply:
Uf, good point